tests/slack/test_plugin.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+
+import responses
+
+from exam import fixture
+from sentry.models import Rule
+from sentry.plugins import Notification
+from sentry.testutils import TestCase
+from sentry.utils import json
+from six.moves.urllib.parse import parse_qs
+
+from sentry_plugins.slack.plugin import SlackPlugin
+
+
+class SlackPluginTest(TestCase):
+    @fixture
+    def plugin(self):
+        return SlackPlugin()
+
+    def test_conf_key(self):
+        assert self.plugin.conf_key == 'slack'
```

Comment:
Should just be:

```python
def test_entry_point(self):
    self.assertAppInstalled('slack', 'sentry_plugins.slack')
    self.assertPluginInstalled('slack', self.plugin)
```

And subclass `PluginTestCase` instead of `TestCase`.